### PR TITLE
Port module stock_picking_invoice_link to v8

### PR DESCRIPTION
**Pull request by alexis-via**
It's just a simple port to v8, without any other changes.

### CODE REVIEW

File: stock_picking_invoice_link/AUTHORS.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 Lorenzo Battistini <lorenzo.battistini@agilebg.com>
```
Comment (pedrobaeza) [inline stock_picking_invoice_link/AUTHORS.txt]: Remove this file and put it on module description.

File: stock_picking_invoice_link/stock.py [hunk 2]
```diff
@@ -26,9 +26,16 @@ class stock_move(orm.Model):
 
     _columns = {
         'invoice_line_id': fields.many2one(
-            'account.invoice.line', 'Invoice line', readonly=True),
+            'account.invoice.line', 'Invoice Line', readonly=True),
```
Comment (pedrobaeza) [inline stock_picking_invoice_link/stock.py:29]: I prefer lower cases in the middle of the sentence, but it's also OK that way. Please add pot file for translating it.

Comment (alexis-via) [inline stock_picking_invoice_link/stock.py:29]: Me too ! But that's how it is everywhere in OpenERP... so I adopted the "OpenERP" convention...

**Comment by eLBati** [comment]
:+1: thanks

**Comment by bealdav** [comment]
:+1:

**Pull request merged**